src/common/error.ts

```diff
@@ -29,6 +29,35 @@ export class UploadError extends Error {
   }
 }
 
+export class NoTokenListed extends Error {
+  /** @internal */
+  constructor() {
+    super(`LIST ERROR: you must list at least one token.`);
+  }
+}
+
+export class NotOwner extends Error {
+  /** @internal */
+  constructor() {
+    super(`LIST ERROR: you should be the owner of the token to list it.`);
```

Comment:
these errors will be used throughout the sdk, so don't hardcode the message. Instead, have a default value, and allow a custom message to be passed in the constructor. Look at the other errors for an example.